database/seeders/IdentitySeeder.php

```diff
@@ -4,221 +4,257 @@
-            [
-                'name' => __('Southeast Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Middle Eastern'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Latin American'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('African'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Women'),
-                'clusters' => ['gender', 'gender-and-sexuality'],
-            ],
-            [
-                'name' => __('Men'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'reachable-when-mixed'],
-            ],
-            [
-                'name' => __('Non-binary people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender non-conforming people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender fluid people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('First Nations'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Inuit'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Métis'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Supporters'),
-                'clusters' => ['lived-experience', 'reachable-when-mixed'],
-            ],
-        ];
+class IdentitySeeder extends Seeder {
+    public function run() {
 
-        foreach ($identities as $identity) {
-            Identity::firstOrCreate([
-                'name' => [
-                    'en' => $identity['name'],
-                    'fr' => trans($identity['name'], [], 'fr'),
-                ],
-                'description' => [
-                    'en' => $identity['description'] ?? null,
-                    'fr' => isset($identity['description']) ? trans($identity['description'], [], 'fr') : null,
-                ],
-                'clusters' => $identity['clusters'] ?? [],
-            ]);
+        // option to use environment to restore or backup to different environment files
+        if (config('seeder.environment') !== null && in_array(config('seeder.environment'), ['production', 'staging', 'local']) === true) {
+            $environment = config('seeder.environment');
+        } else {
+            $environment = config('app.env');
+        }
+
+        // check if there is a JSON file that has stored data for the seeder
+        if (in_array(config('app.env'), ['testing', 'production']) !== true) {
```

Comment:
It seems like this was added to all of the updated seeders, but this prevents the defaults from running in 'testing' and 'production'. However, in most or all cases the default seeders should have been run there. This is what's causing the tests to fail for example.

Reply:
Removed from seeder classes.